source/releasenotes/2024-12-10-drupal-7-d7es.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Tag1 Extended Support module installed on Drupal 7 upstream
+published_date: "2024-12-05"
+categories: [drupal, action-required]
+---
+
+The default Pantheon Drupal 7 upstream has been updated to include the Tag1 Drupal 7 Extended Support (D7ES) module.
```

Comment:
… "required to take advantage of Pantheon's Long Term Support (LTS) for Drupal 7."